### PR TITLE
Cancellation save landing page V1

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
Creates the new landing page in the Membership save journey. This is the initial design so it may change and the buttons will need wired in.

## How to test

new scenario added to storybook as this is not yet working
## Images

image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/helpCentre/HelpCentreEmailAndLiveChat.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ const contactBoxDetailsWideCss = css`
 	}
 `;
 
-const HelpCentreContactBox = (props: HelpCentreContactBoxProps) => {
+export const HelpCentreContactBox = (props: HelpCentreContactBoxProps) => {
```
Comment (andrewHEguardian) [inline client/components/helpCentre/HelpCentreEmailAndLiveChat.tsx:94]: is this used in another component now?

File: client/components/shared/CallCenterEmailAndNumbers.tsx [hunk 2]
```diff
@@ -79,12 +79,16 @@ export interface CallCentreEmailAndNumbersProps extends CallCentreNumbersProps {
...
 	phoneRegionFilterKeys?: PhoneRegionKey[];
 	compactLayout?: boolean;
+	collapsed?: boolean;
 }
 
 export const CallCentreEmailAndNumbers = (
 	props: CallCentreEmailAndNumbersProps,
 ) => {
-	const [indexOfOpenSection, setIndexOfOpenSection] = useState<number>(0);
+	const intailIndex = props.collapsed ? -1 : 0;
```
Comment (andrewHEguardian) [inline client/components/shared/CallCenterEmailAndNumbers.tsx:88]: `initialIndex` typo 🙂

File: client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import {
+	Button,
+	Stack,
+	SvgArrowRightStraight,
+} from '@guardian/source-react-components';
+import { CallCentreEmailAndNumbers } from '../../../shared/CallCenterEmailAndNumbers';
+import { Heading } from '../../shared/Heading';
+import { sectionSpacing } from '../../switch/SwitchStyles';
+
+const buttonLayoutCss = css`
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx:12]: I included this same bit of CSS in `SaveStyles.tsx`

**Review by andrewHEguardian — DISMISSED** [review]
First draft looking good 👍  nicely done

File: client/components/helpCentre/HelpCentreEmailAndLiveChat.tsx (hunk 1 above)
Comment (charleycampbell) [reply client/components/helpCentre/HelpCentreEmailAndLiveChat.tsx:94]: Good spot - no we don't use it now

File: client/components/shared/CallCenterEmailAndNumbers.tsx (hunk 2 above)
Comment (charleycampbell) [reply client/components/shared/CallCenterEmailAndNumbers.tsx:88]: Good spot!

File: client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx (hunk 3 above)
Comment (charleycampbell) [reply client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx:12]: Excellent I've changed the import

**Comment by charleycampbell** [comment]
[Membership Saves - Landing (Page 1) [Initial design]](https://trello.com/c/M9TWmJVs/4036-membership-saves-landing-page-1-initial-design)

**Pull request merged**